README.md

```diff
@@ -23,29 +23,24 @@ We may want to someday write a script that builds this data mapping in a nightly
 
 * In your build.gradle:
 ```groovy
+repositories {
+    maven { url 'https://a8c-libs.s3.amazonaws.com/android' }
+}
 dependencies {
-    implementation 'org.wordpress:libaddressinput.common:0.0.1'
+    implementation 'org.wordpress:libaddressinput.common:0.0.2'
 }
 ```
 
-## Publish an updated version to your local maven repository
-
-You can bump the [version name in the main build file: `common/build.gradle`][1]. After updating the build file, you can build, and publish the library to your local maven repo. That will let you try the new version in your app for example.
-
-```shell
-$ ./gradlew assemble test publishToMavenLocal
-```
-
-## Publish it to Bintray
+## Publish it to S3
 
 When a new version is ready to be published to the remote repository, use the following command to upload it to Bintray:
```

Comment:
s/Bintray/S3/ 🤷

Reply:
@AliSoftware Thanks for catching that! Addressed in https://github.com/wordpress-mobile/libaddressinput/pull/3.